README.md

```diff
@@ -1,4 +1,4 @@
-# ut2019
+# GROUP 5 ut2019
```

Comment:
You are missing navigation tests between the contacts and home page. See for example https://github.com/ut2019-testing/ut2019/blob/c5661c22a9d435235b953bb30d4d4e0ef1d8671c/cypress/integration/test1.spec.js